### ISSUE TITLE
matplotlib not installed in setup.py

### ISSUE THREAD

**Issue by LWprogramming** [patrickvonplaten/controlnet_aux#74]
matplotlib exists in [_deps](https://github.com/patrickvonplaten/controlnet_aux/blob/c8ee2a7e3be86576494a7629cc544bd1790fb179/setup.py#L86) but not in [install_requires](https://github.com/patrickvonplaten/controlnet_aux/blob/c8ee2a7e3be86576494a7629cc544bd1790fb179/setup.py#L167) and extras is empty. Is this intentional? Had to install this by hand in my venv and was kinda surprised

**Comment by patrickvonplaten** [comment]
Yeah that's probably a typo we could remove `matplotlib` from the deps list

**Comment by LWprogramming** [comment]
Ah hmm, I just double-checked the [codebase](https://github.com/search?q=repo%3Apatrickvonplaten%2Fcontrolnet_aux%20matplotlib&type=code) and it looks like it does need it in some files (not just wrapping a numpy call or something)

Checked the two and matplotlib is the only one in _deps that's missing so I'll just add it
